features/toolbar/components/web/ToolbarContainer.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import { ColorPalette } from '../../../base/styles';
+
+import styles from './styles/Styles';
+
+import Icon from 'react-fontawesome';
```

Comment:
I thought you liked grouping the imports by origin? I.e.:

react
react-fontawesome

../../../base/styles

./styles/Styles